gsy_framework/community_datasheet/row_converters.py

```diff
@@ -309,84 +313,74 @@ def _validate_row(cls, row: Dict):
             )
 
 
-class PVRowConverter:
+class LoadRowConverter(AssetsRowConverter):
+    """Convert from the excel row to a grid representation of a Load asset."""
+
+    OPTIONAL_FIELDS = (LoadSheetHeader.DATASTREAM_ID.value,)
+
+    @classmethod
+    def convert(cls, row: Dict) -> Dict:
+        cls._validate_row(row)
+        return {
+            "name": row[LoadSheetHeader.LOAD_NAME.value],
+            "type": "Load",
+            "uuid": str(uuid.uuid4()),
+        }
+
+
+class PVRowConverter(AssetsRowConverter):
     """Convert from the excel row to a grid representation of a PV asset."""
 
-    # These fields are optional because they can be ignored if a PV profile is explicitly provided
     OPTIONAL_FIELDS = (
-        PVSheetHeader.CAPACITY_KW,
-        PVSheetHeader.TILT,
-        PVSheetHeader.AZIMUTH,
-        PVSheetHeader.DATASTREAM_ID,
+        PVSheetHeader.CAPACITY_KW.value,
+        PVSheetHeader.TILT.value,
+        PVSheetHeader.AZIMUTH.value,
+        PVSheetHeader.DATASTREAM_ID.value,
     )
 
     @classmethod
     def convert(cls, row: Dict) -> Dict:
-        """
-        Convert the row parsed from the Excel file to the asset representation used in scenarios.
-        """
         cls._validate_row(row)
         return {
-            "name": row[PVSheetHeader.PV_NAME],
+            "name": row[PVSheetHeader.PV_NAME.value],
```

Comment:
Even though I think that this will not crash anything, isn't PV_NAME a string already? If this is the case, then the `.value` is not needed, same for all other Header class variables.

Reply:
It might be a little bit more confusing, but I am overriting the value of Enum to lower each value so it is clear from the code what each enum means + allowing parsing the column headers in CDS with -- user errors